src/pages/edit_resource/preview_step/PreviewPanel.tsx

```diff
@@ -1,20 +1,35 @@
 import React, { FC } from 'react';
 import { FormikProps, FormikValues, useFormikContext } from 'formik';
-import { StyledContentWrapper, StyledSchemaPartColored } from '../../../components/styled/Wrappers';
 import ResourcePresentation from '../../resource/ResourcePresentation';
 import { Resource } from '../../../types/resource.types';
+import { Box, Typography } from '@material-ui/core';
+import styled from 'styled-components';
+import { useTranslation } from 'react-i18next';
+
+const StyledTypography = styled(Typography)`
+  margin-bottom: 2rem;
+`;
 
 interface DescriptionFieldsProps {
   formikProps: FormikProps<FormikValues>;
 }
 
-const PreviewPanel: FC<DescriptionFieldsProps> = ({ formikProps }) => {
+const PreviewPanel: FC<DescriptionFieldsProps> = () => {
   const { values } = useFormikContext<Resource>();
+  const { t } = useTranslation();
 
   return (
-    <StyledSchemaPartColored color={'#FFFFFF'}>
-      <StyledContentWrapper>{values && <ResourcePresentation resource={values} />}</StyledContentWrapper>
-    </StyledSchemaPartColored>
+    <>
+      <StyledTypography>{t('confirm_before_publishing')}.</StyledTypography>
```

Comment:
jeg pakket inn warningen min i en materia-ui-Alert. Om du  synes min ser grei ut kunne du også brukt samme oppsett men med info 
```
<Alert severity="info">
  <AlertTitle>Info</AlertTitle>
</Alert>
```
![image](https://user-images.githubusercontent.com/46489468/116072451-6467f200-a68f-11eb-9624-9a7d1077ba69.png)

(Fargen min må nok tunes litt)

Reply:
Den ser bra ut, vi kan ta din. Den er kanskje orangen er kanskje pittelitt mørk?

Reply:
Forøvrig, kanskje den orange boksen skulle vært flyttet ned til under stepperen? Da er advarselen nærmere feltene man eventuelt sitter å redigerer i.

Reply:
jeg har også denne: 
![image](https://user-images.githubusercontent.com/46489468/116075303-148b2a00-a693-11eb-8674-906c6a316f34.png)

Reply:
den første orange er mildt sagt mørk ja :-)

Reply:
Ha litt margin til lisens advarselen. Så ville jeg kanskje brukt info istedet for warning på akkurat den.

Reply:
(men jeg har lagt til en informasjons-alert nå da) og stilet den til DLR blå 1.